src/libraries/System.Text.Json/tests/System.Text.Json.SourceGeneration.Unit.Tests/JsonSourceGeneratorDiagnosticsTests.cs

```diff
@@ -50,25 +54,13 @@ public class IndexViewModel
 
             CompilationHelper.RunGenerators(compilation, out var generatorDiags, generator);
 
-            // Expected info logs.
-            string[] expectedInfoDiagnostics = new string[] {
-                "Generated serialization metadata for type System.Collections.Generic.List<ReferencedAssembly.ActiveOrUpcomingEvent>",
-                "Generated serialization metadata for type System.Int32",
-                "Generated serialization metadata for type System.String",
-                "Generated serialization metadata for type System.DateTimeOffset",
-                "Generated serialization metadata for type System.Boolean",
-                "Generated serialization metadata for type ReferencedAssembly.ActiveOrUpcomingEvent",
-                "Generated serialization metadata for type ReferencedAssembly.CampaignSummaryViewModel",
-                "Generated serialization metadata for type JsonSourceGenerator.IndexViewModel",
-            };
-
-            CompilationHelper.CheckDiagnosticMessages(generatorDiags, DiagnosticSeverity.Info, expectedInfoDiagnostics);
-            CompilationHelper.CheckDiagnosticMessages(generatorDiags, DiagnosticSeverity.Warning, new string[] { });
-            CompilationHelper.CheckDiagnosticMessages(generatorDiags, DiagnosticSeverity.Error, new string[] { });
+            CompilationHelper.CheckDiagnosticMessages(DiagnosticSeverity.Info, generatorDiags, Array.Empty<ValueTuple<TextSpan, string>>());
```

Comment:
Nit:
```suggestion
            CompilationHelper.CheckDiagnosticMessages(DiagnosticSeverity.Info, generatorDiags, Array.Empty<(TextSpan, string)>());
```
For this and other uses of `ValueTuple` in tests. Consider labeling elements for better self-documentation.